snuba/subscriptions/data.py

```diff
@@ -163,13 +172,16 @@ def build_request(
                 custom_processing.append(validator.validate)
         custom_processing.append(partial(self.add_conditions, timestamp, offset))
 
+        tenant_ids = {**self.tenant_ids}
+        tenant_ids["referrer"] = referrer
+        if "organization_id" not in tenant_ids:
+            # TODO: Subscriptions queries should have an org ID
+            tenant_ids["organization_id"] = 1
```

Comment:
Is this related with this PR or is it an unrelated bug ?

Reply:
We have no Tenant IDs currently for any subscription query, so for now it's just been Org ID 1 but if we actually wanted to get an insight into which orgs are using how much, we need the Tenant IDs to be propagated through from Sentry instead of just hardcoding values here.

This change is just cleaning this up to now accept org id if it exists (since after [this](https://github.com/getsentry/sentry/pull/57101) and [this](https://github.com/getsentry/snuba/pull/4775) we will start accepting Tenant IDs)